index.html

```diff
@@ -97,6 +97,24 @@
     updatePoints();
 }
 
+/**
+ * Updates the simulation with the value inside the "populationValue" box when "enter" key is pressed.
```

Comment:
This should be more intuitive. How does the user know they have to press enter? Perhaps make it so when they exit the textbox. Better yet, check the value on each input (maybe oninput will help here)